### PR TITLE
Fix floor area ratio

### PR DESCRIPTION
**Pull request by hirao-k**
Fix as https://docs.google.com/spreadsheets/d/1fPY69dLbD6Z9usq1pnGztHTiBMHbKckLxIScq9IIjK0/edit#gid=1888694148

### CODE REVIEW

File: src/app/core/building/building.component.html [hunk 1]
```diff
@@ -511,7 +511,7 @@ <h4>3. 指定容積率と比較し、小さい方を採用</h4>
                 <tbody>
                   <tr>
                     <td>容積率の上限</td>
-                    <td class='text-right'>{{result?.buildingSite?.floorAreaRatioLimit}}%</td>
+                    <td class='text-right'>{{Math.min(result?.buildingSite?.floorAreaRatioLimit * 100, blueprint.floorAreaRatio)}}%</td>
```
Comment (hirao-k) [inline src/app/core/building/building.component.html:514]: @thanhtetaung  Please implement here to work.

Comment (thanhtetaung) [reply src/app/core/building/building.component.html:514]: @k-kin this kind of logic should not be on frontend side. Should return from backend. As I remember Hla Min already told about this previously.

Comment (hirao-k) [reply src/app/core/building/building.component.html]: @thanhtetaung  All right, I add it to response.

Comment (hirao-k) [reply src/app/core/building/building.component.html]: I changed here, so close this conversation

File: src/app/core/building/building.component.html [hunk 2]
```diff
@@ -511,7 +511,7 @@ <h4>3. 指定容積率と比較し、小さい方を採用</h4>
                 <tbody>
                   <tr>
                     <td>容積率の上限</td>
-                    <td class='text-right'>{{result?.buildingSite?.floorAreaRatioLimit}}%</td>
+                    <td class='text-right'>{{result?.buildingSite?.minFloorAreaRatioLimit}}%</td>
```
Comment (hirao-k) [inline src/app/core/building/building.component.html:514]: @thanhtetaung  After I add it API response, I'll tell you that.

Comment (thanhtetaung) [reply src/app/core/building/building.component.html:514]: @k-kin Thanks. After you have finished, I will update Java backend and merge this PR.

Comment (hirao-k) [reply src/app/core/building/building.component.html:514]: @thanhtetaung  I deployed it to API. ( you mentioned not me but kin )

Comment (thanhtetaung) [reply src/app/core/building/building.component.html:514]: @hirao-k Thanks.

**Review by thanhtetaung — APPROVED** [review]

**Pull request merged**